nixos/modules/services/networking/ssh/sshd.nix

```
@@ -272,6 +274,7 @@ in
                 KillMode = "process";
               } // (if cfg.startWhenNeeded then {
                 StandardInput = "socket";
+                StandardError = "journal";
```

Does this also suppress other error message, which should  be not redirected?